### PR TITLE
Implicit DOMTokenList to Seq (e.g. for Element.classList) support

### PR DESCRIPTION
**Pull request by abdolence**
Probably it was accidentally forgotten and it will be convenient to have something like Element.classList.foreach or filter additionally to the others.
Don't you think?

### CODE REVIEW

**Review by armanbilge — APPROVED** [review]

File: src/main/scala/org/scalajs/dom/ext/package.scala [hunk 1]
```diff
@@ -9,6 +9,9 @@ package object ext {
   implicit class PimpedNodeList(nodes: NodeList)
       extends EasySeq[Node](nodes.length, nodes.apply)
 
+  implicit class PimpedDOMTokenList(nodes: DOMTokenList)
```
[the remark this answers is not in the document]
Comment (japgolly) [reply src/main/scala/org/scalajs/dom/ext/package.scala:12]: 100% but let's perform this change on the upcoming 2.x release branch so we don't break compatibility

**Comment by abdolence** [comment]
This PR exists since 2019, probably high time to resolve it somehow?
Just saying :)

**Comment by armanbilge** [comment]
So sorry for that, we are working on it! See https://users.scala-lang.org/t/scala-js-dom-to-become-more-active-2-x-coming-soon/7702

**Comment by abdolence** [comment]
> So sorry for that, we are working on it! See https://users.scala-lang.org/t/scala-js-dom-to-become-more-active-2-x-coming-soon/7702

Can't wait when you release it for Scala 3 as well 🎉 Thanks.

No worries, it is generally just a small inconvenience to copy my implicits from one project to another :)

**Comment by sjrd** [comment]
For 2.x, in #458, I made those things available by default for all `DOMList`s.

**Comment by japgolly** [comment]
Thanks for the PR (and your patience!) @abdolence . This is fixed on `master` by #458 and verified by #546

**Pull request closed without merging**